### ISSUE TITLE
Exponential retry policy takes a parameter "increment_power" but seems to actually mean "increment_base"

### ISSUE THREAD

**Issue by matthchr** [Azure/azure-storage-python#412]
For example:
When setting up a retry policy like so:
```
azure.storage.common.ExponentialRetry(
                initial_backoff=3,
                increment_power=2,
                max_attempts=3)
```
This _sounds_ like it will result in a retry sequence of: `{3, 3 + 1^2, 3 + 2^2}` (since `increment_power` is 2, the powers/exponents are all 2). In reality it generates a sequence `{3, 3 + 2^1, 3 + 2^2}`

Luckily, this is all clarified by the documentation here: https://github.com/Azure/azure-storage-python/blob/9b5e870e615dca75cf9edcf7d68b2cc5f1a61e02/azure-storage-common/azure/storage/common/retry.py#L158

Still, it would be more clear if this were called `increment_base` rather than `increment_power` since the "power"/exponent is actually `n` (number of retries) and is not configurable by the user.

**Comment by zezha-msft** [comment]
Hi @matthchr, thanks for the feedback! And I agree that the argument name should be modified to make it clearer. I'll squeeze in this change in a future release. Thanks!

**Comment by zezha-msft** [comment]
Hi @matthchr, this was fixed [here](https://github.com/Azure/azure-storage-python/commit/d73cd6c173b2c4d6ddbb1ecc30865538ed96908f) and will go out with the next release. Thanks!

**Comment by zezha-msft** [comment]
This was fixed in the latest [release v1.0.0](https://github.com/Azure/azure-storage-python/releases/tag/v1.0.0-common).
